src/pages/Home/components/Map/index.tsx

```diff
@@ -0,0 +1,109 @@
+                '<div style="display:flex;flex-direction:row;align-items:center;">',
+                '<img src="" style="width:68px;height:68px;border-radius:89.5px;"slt="음식점 사진"/>',
+                '<span style="padding-left:10px;font-size:20px;font-weigth:500;text-overflow:ellipsis">', `${selectedMarker.current.title}`, '</span>',
+                '</div>',
+                '<div style="position:relative;left:-125px;top:65px;">',
+                '<div style="border-color:#FF7F23 transparent transparent;border-style:solid;border-width:12px 15px 0;">', '</div>',
+                '<div style="border-color:white transparent transparent;border-style:solid;border-width:12px 15px;position:absolute;top:-1px;">', '</div>',
+                '</div>',
+                '</div>',
+              ].join(''),
+              size: new naver.maps.Size(50, 52),
+              anchor: new naver.maps.Point(25, 26),
+            },
+          });
+        }
+        markerCur.setOptions({
+          icon: {
+            content: [
+              '<div style="display:flex;justify-content:center;text-align:center;border:1px solid #FF7F23;background-color:#FF7F23;border-radius:32.5px;min-width:205px;height:65px;">',
+              '<div style="display:flex;flex-direction:row;align-items:center;">',
+              '<img src="" style="width:68px;height:68px;border-radius:89.5px;"slt="음식점 사진"/>',
+              '<span style="padding-left:10px;font-size:20px;font-weigth:500;text-overflow:ellipsis">', `${item.placeName}`, '</span>',
+              '</div>',
+              '<div style="position:relative;left:-125px;top:65px;">',
+              '<div style="border-color:#FF7F23 transparent transparent;border-style:solid;border-width:12px 15px 0;">', '</div>',
+              '<div style="border-color:#FF7F23 transparent transparent;border-style:solid;border-width:12px 15px;position:absolute;top:-1px;">', '</div>',
+              '</div>',
+              '</div>',
+            ].join(''),
+            size: new naver.maps.Size(50, 52),
+            anchor: new naver.maps.Point(25, 26),
+          },
+        });
+        selectedMarker.current = markerCur;
+        const mapLatLng = new naver.maps.LatLng(item.latitude, item.longitude);
+        mapRef.current.panTo(mapLatLng, 'easeInOutCubic');
+      });
+    };
+    for (let i = 0; i < marker.length; i += 1) {
+      markerRef.current = new naver.maps.Marker({
+        position: new naver.maps.LatLng(marker[i].latitude, marker[i].longitude),
+        title: marker[i].placeName,
+        map: mapRef.current,
+        icon: {
+          content: [
+            '<div style="display:flex;justify-content:center;text-align:center;border:1px solid #FF7F23;background-color:white;border-radius:32.5px;min-width:205px;height:65px;">',
+            '<div style="display:flex;flex-direction:row;align-items:center;">',
+            '<img src={FoodIcon} style="width:68px;height:68px;border-radius:89.5px;"slt="음식점 사진"/>',
```

Comment:
테스트를 하는데 있어 UI가 보이지 않아 불편해요
임시 이미지를 넣어주면 확인하기 쉬울 것 같아요.
```suggestion
//... 상단에서 공용 이미지 import
// import defaultImage from 'assets/images/search/default-image.png';
            `<img src="${defaultImage}" style="width:68px;height:68px;border-radius:89.5px;"slt="음식점 사진"/>`,
```

Reply:
그런데 사진을 넣어보니 사진 비율이 안맞네요. 이부분도 확인해주세요~
<img width="263" alt="image" src="https://user-images.githubusercontent.com/50780281/211348804-478b8b0d-8f9e-4466-85b2-a980549e2424.png">

Reply:
수정했습니다. [0d9b88b](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/50/commits/0d9b88ba5e54423d5a7ecce06c604fb5486934c4)